.gitattributes

```
@@ -1 +1 @@
-*.ksy linguist-language=yaml
+*.ksy linguist-language=yaml eol=lf
```

Looking at various examples of the `eol` attribute on the internet (e.g. [docs.github.com](https://docs.github.com/en/github/using-git/configuring-git-to-handle-line-endings#example) or [git-scm.com](https://git-scm.com/docs/gitattributes#_end_of_line_conversion)), it seems to me that the `eol` attribute is used only with `text`, not only by itself. Probably makes sense to add it?

```suggestion
*.ksy linguist-language=yaml text eol=lf
```